commanderbot_ext/ping/ping_cog.py

```diff
@@ -0,0 +1,13 @@
+import discord
+from commanderbot_lib.logging import Logger, get_clogger
+from discord.ext.commands import Bot, Cog, Context, command
+
+
+class PingCog(Cog):
+    def __init__(self, bot: Bot):
+        self.bot: Bot = bot
+        self._log: Logger = get_clogger(self)
```

Comment:
I don't see `self._log` actually being used anywhere here, so it should be removed along with the unused imports that follow.

Reply:
Fixed in latest commit